.github/workflows/CICD.yml

```diff
@@ -1208,3 +1208,19 @@ jobs:
         lima bash -c "cd work && cargo test --features 'feat_selinux'"
     - name: Lint with SELinux
       run: lima bash -c "cd work && cargo clippy --all-targets --features 'feat_selinux' -- -D warnings"
+
```

Comment:
that's up to you whether you want to leave this or not. I realized after adding this that the i686 builds currently also fail with the same error. (with current code, stdbuf is compiled with 32 bits architecture, but libstdbuf is compiled with 64 bits architecture).